Exoft.Gamification.Api.Data.Core/Interfaces/IAchievementRepository.cs

```diff
@@ -8,7 +8,7 @@ namespace Exoft.Gamification.Api.Data.Core.Interfaces
 {
     public interface IAchievementRepository : IRepository<Achievement>
     {
-        Task<Achievement> IsUserHaveAchievementAsync(Guid userId, Guid achievementId);
-        Task<ReturnPagingInfo<Achievement>> GetPagedAchievementByUserAsync(PagingInfo pagingInfo, Guid userId);
+        Task<Achievement> DoesUserHaveAchievementAsync(Guid userId, Guid achievementId);
```

Comment:
DoesUserHaveAchievement - there I just correct you from English language perspective....

DoesUserHaveAchievementAsync - just translate this name into ukrainian... is it correct name for this method?